python/DEV_SETUP.md

```diff
@@ -71,17 +61,34 @@ you may need to run `~/.local/bin/poetry install` and `~/.local/bin/poetry shell
 instead. You can fix this by adding `export PATH="$HOME/.local/bin:$PATH"` to
 your `~/.bashrc` and closing/re-opening the terminal.\_
 
-## Using Poetry
+## Using uv
 
-Poetry allows to use SK from the local files, without worrying about paths, as
+uv allows us to use SK from the local files, without worrying about paths, as
 if you had SK pip package installed.
 
-To install Poetry in your system, first, navigate to the directory containing
-this README using your chosen shell. You will need to have Python 3.10, 3.11, or 3.12
-installed.
+To install uv in your system, first, navigate to the directory containing
```

Comment:
We mention installation here and there is another section below that lists the command to install uv.